syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode 
{
    int32 error_code = 1;
    string error_msg = 2;
    bool success = 3;
}
message LoginRequest 
{
    string username = 1;
    string password = 2;
}

message LoginResponse 
{
    ResultCode result_code = 1;
    bool success = 2;
}

service UserServiceRpc
{
    rpc Login(LoginRequest) returns (LoginResponse) {}
}
